pkg/namesgenerator/names-generator.go

```diff
@@ -569,6 +578,9 @@ var (
 
 		// Ada Yonath - an Israeli crystallographer, the first woman from the Middle East to win a Nobel prize in the sciences. https://en.wikipedia.org/wiki/Ada_Yonath
 		"yonath",
+
+		// Konrad Zuse - a German engineer, inventor of the first programmable computer. https://en.wikipedia.org/wiki/Konrad_Zuse
```

Comment:
@thaJeztah This might be controversial, However he wasn't directly a member of the nazi party and he invented the first programmable computer.
What do other people from docker think about this?